client/src/containers/AddressPage.tsx

```diff
@@ -260,7 +265,11 @@ export default class AddressPage extends Component<AddressPageProps, State> {
               <PropertiesList
                 state={state}
                 send={send}
-                onOpenDetail={this.handleAddrChange}
+                onOpenDetail={(bbl: string) => {
+                  this.handleAddrChange(bbl);
+                  logAmplitudeEvent("addressChangePortfolio", analyticsEventData);
```

Comment:
do we want the granularity of knowing that the address changed from a click on the address vs the bbl vs the view details button?

Reply:
oh right, thanks, yeah that'd be good to know! Just updated to have the column name as a property on the event.